test/distro/distroSpec.js

```diff
@@ -9,9 +9,12 @@ const DIST_DIR = path.join(__dirname, '../../dist');
 
 describe('modules', function() {
 
-  it('should expose CJS bundle', function() {
+  it.skip('should expose CJS bundle', function() {
```

Comment:
Is this `skip` a left over, given that we shimmed `window`, too?

Reply:
It won't work even with shimmed window but the error message refers to `import` being used in `bpmn-js`. I decided to leave it `skip`ped as I thought maybe some day we want to fix that.

Reply:
Let us remove the shim all-together then.

Reply:
Done.